src/query/service/src/pipelines/builders/builder_window.rs

```diff
@@ -179,34 +182,32 @@ impl PipelineBuilder {
         })?;
 
         let operator = DataOperator::instance().operator();
+
         let location_prefix =
             query_spill_prefix(self.ctx.get_tenant().tenant_name(), &self.ctx.get_id());
+
+        let disk_spill =
+            CacheManager::instance()
```

Comment:
CacheManager is used to cache data read from s3.

We need introduce another config instead of reusing cache.

```
[spill]
local_disk_path = "./.databend/_query_spill/"  ## enable if it's not empty
local_disk_ratio = 60   ## only write 60% into disk otherwise spill to remote storage
```